docs/reference/strava_model.rst

```diff
@@ -1,4 +1,72 @@
-.. automodule:: stravalib.strava_model
-   :members:
-   :undoc-members:
-   :show-inheritance:
+============================
+Strava_model Module
+============================
+.. currentmodule:: stravalib.strava_model
+
+The `strava-model` module provides... stuff and lots of it
+
+`TODO: possibly organize by activity, club etc?`
+
+Summary Classes
+----------------
```

Comment:
Did you add all of these manually? If so, it seems that `Activity` is missing and leading to warnings while generating the docs:

```
stravalib/docs/autodoc_pydantic.rst: WARNING: JSON schema can't be generated for 'stravalib.model.BaseEffort' because the following pydantic fields can't be serialized properly: ['activity']
```

Reply:
@jsamoocha i did add these manually. there's not a great way to automate what i setup BUT i can absolutely fix it! i'll look again at all of this later in the week and will see what those errors are. sphinx always raises warnings but i missed the errors! thank you for catching all of this.

Reply:
i'm not seeing this error anymore. Activity wasn't missing but i also found some other issues with docstrings that had unusual spacing and indentation. could you see if you're still seeing strange behavior?

Reply:
All errors and warnings are gone now, great!